src/components/DeploymentDetails.vue

```diff
@@ -107,6 +107,9 @@
   const api = useWorkspaceApi()
   const updateScheduleLoading = ref(false)
 
+  const { workPool } = useWorkPool(props.deployment.workPoolName ?? '')
+  const isPushWorkPool = computed(() => workPool.value?.isPushPool)
```

Comment:
Nit: we really don't need a computed for this. `workPool.isPushPool` can be used directly